gnuradio-runtime/lib/hier_block2_detail.cc

```diff
@@ -749,11 +754,8 @@ void hier_block2_detail::flatten_aux(flat_flowgraph_sptr sfg) const
     }
     */
 
-    // Construct unique list of blocks used either in edges, inputs,
-    // outputs, or by themselves.  I still hate STL.
```

Comment:
This change, and the line deletion below, look unrelated to this PR.

Reply:
I agree. Thanks for suggestion.
I'll force-push updated version soon.